src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/TensorsInfo.cs

```diff
@@ -315,7 +385,15 @@ protected virtual void Dispose(bool disposing)
             if (_handle != IntPtr.Zero)
             {
                 NNStreamerError ret = NNStreamerError.None;
-                ret = Interop.Util.DestroyTensorsInfo(_handle);
+                try
+                {
+                    ret = Interop.Util.DestroyTensorsInfo(_handle);
+                }
+                catch (DllNotFoundException e)
```

Comment:
DllNotFound is already checked in CheckNNStreamerSupport()

Reply:
ACK